README.md

```diff
@@ -747,6 +748,38 @@ This is how it can be included in your `configuration.nix`:
 }
 ```
 
+## Use with home manager
+
+sops-nix also provides a home-manager module.
+This module provides a subset of features provided by the system-wide sops-nix since features like the creation of the ramfs and changing the owner of the secrets are not available for non-root users.
+
+Instead of running as an activation script, sops-nix runs as a systemd user service called `sops-nix.service`.
+And instead of decrypting to `/run/secrets`, the secrets are decrypted to `$XDG_RUNTIME_DIR/secrets`.
+
+Usage example:
+```nix
+{
+  # NixOS home-manager configuration
+  home-manager.sharedModules = [
+    /path/to/sops-nix/modules/home-manager/sops.nix
+  ];
+
+  # Configuration via home.nix
+  imports = [
+    /path/to/sops-nix/modules/home-manager/sops.nix
+  ];
+
+  # Configuration of secrets
+  sops = {
+    age.sshKeyPaths = [ "/home/user/path-to-ssh-key" ]; # must have no password!
```

Comment:
I am inclined to make normal age keys the default here as a best practice. Most of the time you want to have ssh keys encrypted because of their actual use.